msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/AccountState.kt

```diff
@@ -98,6 +100,19 @@ class AccountState private constructor(
         }
     }
 
+    private fun addDefaultScopes(scopes: List<String>?): List<String> {
```

Comment:
Why is this done here, rather than inside the controller? The latter is what MSAL is doing currently, and wherever possible I would like to stick to existing designs/architectures.

Reply:
This function wasn't used and has been removed.

Reply:
That's not the question. I can see default scopes are added in the interface layer (in `getAccessTokenInternal()`), rather than inside the controller. Why not the latter, and align with how MSAL's browser-delegated flow operates?

Reply:
I have changed how the `getAccessTokenInternal()` is called from the old deprecated method and then removed the merging of scopes.

Reply:
I have changed the code in getAccessMethod so that empty scopes are not passed to `getAccessTokenInternal` obviating the need to merge scopes.